packages/components/Notification/src/NotificationTokens.ios.ts

```diff
@@ -1,7 +1,85 @@
+import { Theme } from '@fluentui-react-native/framework';
+import { TokenSettings } from '@fluentui-react-native/use-styling';
+import { NotificationTokens } from './Notification.types';
 import { DynamicColorIOS } from 'react-native';
 
+export const defaultNotificationTokens: TokenSettings<NotificationTokens, Theme> = (t: Theme) =>
+  ({
+    backgroundColor: t.colors.background,
+    borderColor: 'transparent',
+    borderRadius: 12,
+    borderWidth: 1,
+    color: t.colors.brandForeground1,
+    fontFamily: 'primary',
+    fontLetterSpacing: -0.24, // iOS only prop
+    fontLineHeight: 20,
+    fontSize: 15,
+    fontWeight: '600',
+    padding: 16,
+    hasTitle: {
+      fontLetterSpacing: -0.08, // iOS only prop
+      fontLineHeight: 18,
+      fontSize: 13,
+      fontWeight: '400',
+      paddingVertical: 12,
+    },
+    isBar: {
+      borderRadius: 0,
+      fontWeight: '400',
+    },
+    primary: {
+      backgroundColor: notificationColors.brandBackground4,
+      color: notificationColors.brandForeground4,
+      // color: '#0F6CBD',
+      disabledColor: notificationColors.brandForegroundDisabled,
+      /**
+       * None of the foreground tokens here have pressed versions so the foreground color with an alpha value was used.
+       * The FluentUI Apple NotificationView was used to color match.
+       */
+      pressedColor: '#0F6CBD30',
```

Comment:
Could we either use `rgba(15, 108, 189, 0.19)` instead of the hex value, or leave a comment besides the opacity in decimal? I mostly wanted to highlight that the opacity of 20% is something we guesstimated by comparing with the native button.